glossar/README.md

```diff
@@ -101,6 +101,8 @@ Jeder, der ein Geschäft betreibt und dafür eine Registrierkasse benutzt ist f
 
 Der Kassenbetreiber betreibt die fiskaltrust.Middleware lokal an seinem Standort.
```

Comment:
"Jeder, der ein Geschäft betreibt und dafür ein elektronisches Aufzeichnungssystem (Kasse) benutzt ist für fiskaltrust ein Kassenbetreiber. Wenn ...
... Der Kassenbetreiber betreibt die fiskaltrust.Middleware in seiner ´sicheren Einsatzumgebung´.  Bei lokalen Systemen ist diese der Standort (Outlet), bei Cloud-Kassen kann das auch das Rechenzentrum sein, in dem die Cloud-Kasse betrieben wird."